x-pack/plugins/spaces/public/management/edit_space/enabled_features/feature_table.tsx

```diff
@@ -20,15 +32,175 @@ interface Props {
+            <EuiCheckbox {...checkboxProps} />
+          </EuiFlexItem>
+          {category.euiIconType ? (
+            <EuiFlexItem grow={false}>
+              <EuiIcon size="m" type={category.euiIconType} />
+            </EuiFlexItem>
+          ) : null}
+          <EuiFlexItem grow={1}>
+            <EuiTitle size="xs">
+              <h4 className="eui-displayInlineBlock">{category.label}</h4>
+            </EuiTitle>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      );
 
-    return <EuiInMemoryTable columns={this.getColumns()} items={items} />;
+      // TODO: Enable once https://github.com/elastic/eui/issues/3881 is resolved (will land in version 28.4.0)
+      // const label: string = i18n.translate('xpack.spaces.management.featureAccordionSwitchLabel', {
+      //   defaultMessage: '{enabledCount} / {featureCount} features visible',
+      //   values: {
+      //     enabledCount,
+      //     featureCount,
+      //   },
+      // });
+      // const extraAction = (
+      //   <EuiText size="s" aria-hidden="true" color={'subdued'}>
+      //     {label}
+      //   </EuiText>
+      // );
+
+      const accordian = (
+        <EuiAccordion
+          id={`featureCategory_${category.id}`}
+          key={category.id}
+          arrowDisplay="right"
+          buttonContent={buttonContent}
+          // extraAction={extraAction}
+        >
+          <div className="spcFeatureTableAccordionContent">
+            <EuiSpacer size="s" />
+            {featuresInCategory.map((feature) => {
+              const featureChecked = !(
+                space.disabledFeatures && space.disabledFeatures.includes(feature.id)
+              );
+
+              return (
+                <EuiFlexGroup key={`${feature.id}-toggle`}>
+                  <EuiFlexItem grow={false}>
+                    <EuiCheckbox
+                      id={`featureCheckbox_${feature.id}`}
+                      data-test-subj={`featureCheckbox_${feature.id}`}
+                      checked={featureChecked}
+                      onChange={this.onChange(feature.id) as any}
+                      label={feature.name}
+                    />
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+              );
+            })}
+          </div>
+        </EuiAccordion>
+      );
+
+      accordians.push({
+        order: category.order ?? Number.MAX_SAFE_INTEGER,
+        element: accordian,
+      });
+    });
+
+    accordians.sort((a1, a2) => a1.order - a2.order);
+
+    const featureCount = this.props.features.length;
+    const enabledCount = getEnabledFeatures(this.props.features, this.props.space).length;
+    const controls = [];
+    if (enabledCount < featureCount) {
+      controls.push(
+        <EuiLink onClick={() => this.showAll()} data-test-subj="showAllFeaturesLink">
+          <EuiText size="xs">
+            {i18n.translate('xpack.spaces.management.showAllFeaturesLink', {
+              defaultMessage: 'Show all',
```

Comment:
Nit: Maybe "Select all" (and "Deselect all" below) would be better wording?

Reply:
I like it!